docs/resources/dds_instance.md

```diff
@@ -157,6 +157,8 @@ The following arguments are supported:
 
 **NOTE:** The instance will be restarted in the background when switching SSL. Please operate with caution.
 
+* `enabled` - (Optional, Bool) Specifies whether to enable second level monitoring.
```

Comment:
this parameter name should be change, we usually use it to indicate whether the instance is enabled

Reply:
second_level_monitoring_enabled